### ISSUE TITLE
Error fetching data. 

### ISSUE THREAD

**Issue by gccousens** [euroargodev/argopy#75]
Hello, I'm using argopy for the first time and I'm having trouble fetching data. The error seems to be with the ERDDAP data fetcher. I'm not an experienced coder, but I'm not sure what I'm doing wrong. I am working in Jupyter notebook. 

#### MCVE Code Sample

```python
from argopy import DataFetcher as ArgoDataFetcher
ds = ArgoDataFetcher().region([-75, -45, 20, 30, 0, 100, '2011', '2012']).to_xarray()
```

#### Expected Output
Expected to be able to print(ds) and see the data array. 

#### Problem Description

The code fails on the second line and I get the following error: 
InvalidFetcher: Requested data fetcher 'erddap' not available ! Please try again with any of: localftp
argovis

#### Versions

<details><summary>Output of `argopy.show_versions()`</summary>

</details>

**Comment by gmaze** [comment]
Hi @gccousens 
Can you please post:
- the full stack of error and 
- the output of the `argopy.show_versions()` command ?

**Comment by gccousens** [comment]
Hi @gmaze 

**Here are the warnings I get when I `import argopy` :**
UserWarning: An error occured while loading the ERDDAP data fetcher, it will not be available !
<class 'ModuleNotFoundError'>
No module named 'erddapy.utilities'
  warnings.warn("An error occured while loading the ERDDAP data fetcher, "
UserWarning: An error occured while loading the ERDDAP index fetcher, it will not be available !
<class 'ModuleNotFoundError'>
No module named 'erddapy.utilities'
  warnings.warn("An error occured while loading the ERDDAP index fetcher, "

**Here is the full error from the code in my original comment:**
InvalidFetcher                            Traceback (most recent call last)
<ipython-input-2-52b7d204dc29> in <module>
      1 from argopy import DataFetcher as ArgoDataFetcher
----> 2 ds = ArgoDataFetcher().region([-75, -45, 20, 30, 0, 100, '2011', '2012']).to_xarray()

~\anaconda3\lib\site-packages\argopy\fetchers.py in __init__(self, mode, src, ds, **fetcher_kwargs)
     74         # Load data source access points:
     75         if self._src not in AVAILABLE_DATA_SOURCES:
---> 76             raise InvalidFetcher("Requested data fetcher '%s' not available ! Please try again with any of: %s"
     77                                  % (self._src, "\n".join(AVAILABLE_DATA_SOURCES)))
     78         else:

InvalidFetcher: Requested data fetcher 'erddap' not available ! Please try again with any of: localftp
argovis

**Here is the output of `argopy.show_versions()` :**
UserWarning: Distutils was imported before Setuptools. This usage is discouraged and may exhibit undesirable behaviors or errors. Please use Setuptools' objects directly or at least import Setuptools first.
  warnings.warn(

INSTALLED VERSIONS

commit: None
python: 3.8.3 (default, Jul  2 2020, 17:30:36) [MSC v.1916 64 bit (AMD64)]
python-bits: 64
OS: Windows
OS-release: 10
machine: AMD64
processor: Intel64 Family 6 Model 69 Stepping 1, GenuineIntel
byteorder: little
LC_ALL: None
LANG: None
LOCALE: English_United States.1252
libhdf5: 1.10.4
libnetcdf: 4.7.3

argopy: 0.1.6
xarray: 0.16.2
pandas: 1.0.5
numpy: 1.18.5
scipy: 1.5.0
netCDF4: 1.5.3
pydap: None
h5netcdf: None
h5py: 2.10.0
Nio: None
zarr: None
cftime: 1.3.0
nc_time_axis: None
PseudoNetCDF: None
rasterio: None
cfgrib: None
iris: None
bottleneck: 1.3.2
dask: 2.20.0
distributed: 2.20.0
matplotlib: 3.2.2
cartopy: 0.18.0
seaborn: 0.10.1
numbagg: None
setuptools: 49.2.0.post20200714
pip: 20.1.1
conda: 4.9.2
pytest: 5.4.3
IPython: 7.16.1
sphinx: 3.1.2

I installed the package using the conda-forge channel in my base environment and then I tried again in a new environment and got the same errors. `conda install -c conda-forge argopy`

Thank you for your help!

**Comment by gmaze** [comment]
ok, this happens because you're using the last version of the erddapy dependency with a version of argopy that is no longer compatible with it 

you have 2 options to solve this:
- wait for the next release of argopy (sometimes in December)
- install the last dev version of argopy with pip:
```bash
pip install git+http://github.com/euroargodev/argopy.git@master
```

**Comment by ocefpaf** [comment]
You can also downgrade your erddapy with:

```
conda install erddapy==0.5.3
```

PS: I'm fixing the argopy conda package now!

**Comment by gccousens** [comment]
@gmaze @gmaze 

Thank you for your help! I had the same issues when I downgraded erddapy after the conda-forge install. I uninstalled the packaged and reinstalled with pip and it worked! 
```python
pip install git+http://github.com/euroargodev/argopy.git@master
```

So now it works at the command line. I haven't gotten it to work in Jupyter notebooks where I'm working. I'm not sure the notebook is finding the pip install or something... but that's a different problem. Thanks again!
